src/extension.ts

```diff
@@ -257,6 +263,19 @@ function getSchemaAssociations(): ISchemaAssociation[] {
   return associations;
 }
 
+function sendStartupTelemetryEvent(telemetry: TelemetryService, initialized: boolean, err?: Error): void {
+  const startUpEvent = {
+    name: 'startup',
+    properties: {
+      'yaml.server.initialized': initialized,
+    },
+  };
+  if (err) {
+    startUpEvent.properties['error'] = err;
```

Comment:
it doesn't work unless err is a string. You need to check if err.message exists and send it